src/main/java/uk/gov/hmcts/cmc/claimstore/models/ccj/RepaymentPlan.java

```diff
@@ -17,15 +22,32 @@
         EVERY_MONTH
     }
 
-    private final double remainingAmount;
-    private final double firstPayment;
-    private final double instalmentAmount;
+    @Valid
```

Comment:
`@Valid` is needed only if you want to validate nested properties of the object, it can be removed. Please apply to other places you've modified in this PR as well.

Reply:
done